docs/2024/README.md

```diff
@@ -181,7 +181,7 @@ For any inquiries contact [HAAM Community](mailto:haamcommunity2023@gmail.com) o
 - **Lecture** Introduction to Population Genetics: _Stephan Schiffels_
 - **Practical** Data Curation: Poseidon: _Clemens Schmid_
 - **Practical** Unsupervised Analyses: PCA/MDS, Admixture: _Jazmín Ramos-Madrigal_
-- **Round table** _Jazmín Ramos-Madrigal_
+- **Round table** Unsupervised clustering analyses & Ancient DNA: _Jazmín Ramos-Madrigal_
```

Comment:
This should be moved to the practical sessions. Jazmín joined for the roundtable as well, but she ran a proper practical session, which is where the slides are from.